runner.go

```diff
@@ -822,11 +822,16 @@ func atomicWrite(path string, contents []byte, perms os.FileMode, backup bool) e
 		}
 	}
 
-	f, err := ioutil.TempFile(parent, "")
+	d, err := ioutil.TempDir(os.TempDir(), "")
```

Comment:
I think it's more idiomatic to use `""` here instead of `os.TempDir`, which will default to tmpdir anyway